osmaxx/countries/migrations/0001_initial.py

```diff
@@ -1,26 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from django.db import migrations, models
-import osmaxx.countries.storage
-import osmaxx.countries.fields
-import django.contrib.gis.db.models.fields
-
-
-class Migration(migrations.Migration):
-
-    dependencies = [
-    ]
-
-    operations = [
-        migrations.CreateModel(
```

Comment:
more precisely: dropping the schema manually on production instances only, not in general.